src/components/Title/Title.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+class Title extends React.Component {
+    render(){
+        return (
+            <div className="title-container">
+               <h3>I am the Title</h3>
+               <div className="heading-container">
+                   <h3>I am the heading</h3>
+               </div>
+                <h4 className="instruction">Choose your favorite</h4>
```

Comment:
Shouldn't this just go in the previous heading where you've written 'I am the heading'? Seems unnecessary to have 3 headings